XML/TestCases/PerformanceTests.xml

```diff
@@ -281,6 +281,7 @@
         <setupType>OneVM12Disk</setupType>
         <AdditionalHWConfig>
             <DiskType>Managed</DiskType>
+            <SCType>Premium</SCType>
```

Comment:
So, if user wants to run the FIO test on Standard storage, how does he do that?

Reply:
@iamshital how about to use a replaceable parameter?

Reply:
it doesn't make sense to run fio test on Standard storage, from the point of testing Linux image

Reply:
@LiliDeng Yes. Sounds good. 
@lizzha , I understand that, running standard storage is not required **now**, but my recommendation is - LISAv2 should be able to handle user scenario without making any change in code.

For example.,

We could add below tag in Test Definition
```
<StorageAccountType>PERF_STORAGE_ACCOUNT_TYPE</StorageAccountType>
```
And create a replaceable parameter : 

```
	<Parameter>
		<ReplaceThis>PERF_STORAGE_ACCOUNT_TYPE</ReplaceThis>
		<ReplaceWith>Premium_LRS</ReplaceWith>
	</Parameter>
```

And then follow Lili's flow to switch to storage account for only that test case.

Reply:
> @LiliDeng Yes. Sounds good.
> @lizzha , I understand that, running standard storage is not required **now**, but my recommendation is - LISAv2 should be able to handle user scenario without making any change in code.
> 
> For example.,
> 
> We could add below tag in Test Definition (AdditionalHWConfig)
> 
> ```
> <StorageAccountType>PERF_STORAGE_ACCOUNT_TYPE</StorageAccountType>
> ```
> 
> And create a replaceable parameter :
> 
> ```
> 	<Parameter>
> 		<ReplaceThis>PERF_STORAGE_ACCOUNT_TYPE</ReplaceThis>
> 		<ReplaceWith>Premium_LRS</ReplaceWith>
> 	</Parameter>
> ```
> 
> And then follow Lili's flow to switch to storage account for only that test case.

Thanks, will update by your advice and need more testing.